graphics/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//flutter/common/config.gni")
+import("//flutter/testing/testing.gni")
+
+source_set("graphics") {
+  sources = [
+    "gl_context_switch.cc",
+    "gl_context_switch.h",
+    "persistent_cache.cc",
+    "persistent_cache.h",
+    "texture.cc",
+    "texture.h",
+  ]
+
+  deps = [
+    "//flutter/assets",
+    "//flutter/fml",
+    "//flutter/shell/version:version",
+    "//third_party/rapidjson",
+    "//third_party/skia",
+  ]
```

Comment:
Shouldn't this have `public_configs = [ "//flutter:config" ]` like most other targets?